app/services/foreman/plugin.rb

```diff
@@ -93,6 +93,17 @@ def installed?(id)
 
     def initialize(id)
       @id = id.to_sym
+      add_loggers
+    end
+
+    def add_loggers
+      return unless (defined?(Foreman::Logging) && SETTINGS[@id])
+
+      if (loggers = SETTINGS[@id][:loggers])
```

Comment:
For Foreman core, I considered putting the configuration into the settings.yaml file, but due to the size of the default configuration I thought a separate yaml file more consumable (and not crowd the settings.yaml file). If folks would prefer I can rework this to have `:logging:` node in settings.yaml with what is in logging.yaml now.

When you say default configuration - are you referring to default configurations for plugins or for logging in general?